components/time/src/provider/mod.rs

```diff
@@ -136,7 +136,11 @@ impl AsULE for VariantOffsetsWithMetazoneMembershipKind {
     fn from_unaligned([std, dst]: Self::ULE) -> Self {
         Self {
             offsets: VariantOffsets {
-                standard: UtcOffset::from_seconds_unchecked(
+                standard: UtcOffset::from_seconds_unchecked(if std == i8::MAX {
+                    // Special bit pattern for value that appears in TZDB but is not
+                    // representable by our schema.
+                    -2670
```

Comment:
This is the correct fix if we care about the extra 30 seconds from 53 years ago. Do we care about that? It's a legacy that I would prefer to leave behind. I think it will fall back to a localized GMT format, which seems fine.